sssd_test_framework/utils/tools.py

```diff
@@ -599,54 +599,64 @@ def __init__(self, host: MultihostHost) -> None:
         """
         super().__init__(host)
 
-    def passwd(self, name: str | int) -> PasswdEntry | None:
+    def passwd(self, name: str | int, service=None) -> PasswdEntry | None:
         """
         Call ``getent passwd $name``
 
-        :param name: User name or id.
+        :param name: Username or id.
```

Comment:
I don't disagree with this change, but it is not related to this commit, please move it to standalone commit if you want to keep it. Same on other places.